consensus/simtests/src/tests/simtests.rs

```diff
@@ -119,6 +124,243 @@ mod test {
         );
     }
 
+    // Tests the fast path transactions. It produces a defined number of transactions and it randomizes the outcome (accept or reject). Then the nodes
+    // are voting on the rejections with priority in order to force the transactions to be rejected. The rest should get accepted. The rest is verified
+    // based on the commit output.
+    #[sim_test(config = "test_config()")]
+    async fn test_committee_fast_path() {
+        telemetry_subscribers::init_for_testing();
+        let db_registry = Registry::new();
+        DBMetrics::init(RegistryService::new(db_registry));
+
+        const NUM_OF_AUTHORITIES: usize = 10;
+        const REJECTION_PROBABILITY: f64 = 0.35;
+        const NUM_TRANSACTIONS: u16 = 5000;
+        const TRANSACTIONS_BATCH_SIZE: u16 = 5;
+
+        let (committee, keypairs) = local_committee_and_keys(0, [1; NUM_OF_AUTHORITIES].to_vec());
+        let protocol_config = ProtocolConfig::get_for_max_version_UNSAFE();
+        std::env::set_var("SIMTEST_CONSENSUS_GC_DEPTH_OVERRIDE", "3");
+
+        let mut authorities = Vec::with_capacity(committee.size());
+        let mut transaction_clients = Vec::with_capacity(committee.size());
+        let mut boot_counters = [0; NUM_OF_AUTHORITIES];
+        let mut clock_drifts = [0; NUM_OF_AUTHORITIES];
+        clock_drifts[0] = 50;
+        clock_drifts[1] = 100;
+        clock_drifts[2] = 120;
+
+        struct FPTransactionVerifier {}
+
+        impl TransactionVerifier for FPTransactionVerifier {
+            fn verify_batch(&self, _transactions: &[&[u8]]) -> Result<(), ValidationError> {
+                Ok(())
+            }
+
+            fn verify_and_vote_batch(
+                &self,
+                batch: &[&[u8]],
+            ) -> Result<Vec<TransactionIndex>, ValidationError> {
+                let mut rejected_indices = vec![];
+
+                // Randomly decide which transactions to reject according to the rejection probability.
+                for (index, transaction) in batch.iter().enumerate() {
+                    let rejected = rand::thread_rng().gen_bool(REJECTION_PROBABILITY);
+                    if rejected {
+                        tracing::info!(
```

Comment:
Remove this log line? Info is too verbose and transaction[0] is not the digest.

Reply:
Shall I reduce to debug/trace instead for now?

Reply:
If you think this is useful, can you fix the message too?